src/js/themes/hpe.js

```diff
@@ -2138,6 +2138,23 @@ const buildTheme = (tokens, flags) => {
           color: { light: 'rgb(245, 245, 245)', dark: 'rgb(44, 44, 44)' },
         },
       },
+      extend: ({ disabled, theme }) => `
+        &::before {
+          display: block;
+          position: absolute;
+          content: '';
+          width: 3px;
```

Comment:
is there not a token for `3px` ? or some size we can use here?

Reply:
Done. Using a primitive value from our scale since there are not semantic tokens that would be appropriate.